Server/src/Config.cpp

```diff
@@ -592,17 +643,12 @@ void Config::parseTopics(const YAML::Node &node) {
         for (YAML::const_iterator it = node["names"].begin(); it != node["names"].end(); ++it) {
             try {
                 // Only add topic names that are initialized, otherwise ignore them
-                if (topic_names_map.find(it->first.as<std::string>()) != topic_names_map.end()) {
-                    if (it->second.Type() == YAML::NodeType::Null) {
-                        topic_names_map[it->first.as<std::string>()] = "";
-                    } else {
-                        topic_names_map[it->first.as<std::string>()] = it->second.as<std::string>();
-                    }
-                } else if (debug_general)
+                if (topic_names_map.find(it->first.as<std::string>()) != topic_names_map.end())
```

Comment:
I don't believe you intended to change this.  This can happen if your fork isn't up-to-date with the master.    In order to maintain your fork, you need to periodically sync it to the master/upstream.  See this link for details on how to do that: https://help.github.com/articles/syncing-a-fork/

For now, I can fix this on merge.